compiler/rustc_apfloat/src/ieee.rs

```diff
@@ -2412,7 +2412,7 @@ mod sig {
         let _: Loss = shift_right(&mut dst[..dst_limbs], &mut 0, shift);
 
         // We now have (dst_limbs * LIMB_BITS - shift) bits from `src`
-        // in `dst`.  If this is less that src_bits, append the rest, else
+        // in `dst`. If this is less that src_bits, append the rest, else
```

Comment:
Please back out changes to `compiler/rustc_apfloat`, we don't want these to diverge again:
* in-tree: https://github.com/rust-lang/rust/commits/master/compiler/rustc_apfloat
* in-progress extraction: https://github.com/LykenSol/rustc_apfloat-git-history-extraction/commits/main